src/types/homepage.ts

```diff
@@ -91,27 +104,26 @@ export type HomepageEditorHeroSection =
   | EditorHeroHighlightsSection
 
 export type HeroFrontmatterSection = { hero: HomepageEditorHeroSection }
-// TODO: add properties here instead of typing as `Record<string, unknown>`
-// we can find them in `HomepagePreview`
-export type ResourcesFrontmatterSection = { resources: Record<string, unknown> }
+export type ResourcesFrontmatterSection = { resources: ResourcesSection }
```

Comment:
sanity check here, we are certain about the shape for this ya?

Reply:
This + all the rest of the `infopic` + `infobar`, etc

Reply:
Yup, it should follow the frontmatter format already used in the homepage, but for sanity check let me confirm with @seaerchin.

Reply:
i _think_ so? but this was done quite awhile back tbh... if there are no build issues, it shuold be ok cos i remember i used `Record<string, unknown>` to avoid having to trace the properties (for velocity)